packages/perseus/src/widgets/radio/choice.jsx

```diff
@@ -9,192 +9,119 @@ import {Popover, PopoverContent} from "@khanacademy/wonder-blocks-popover";
-        }),
-        multipleSelect: PropTypes.bool,
-        checked: PropTypes.bool,
-        className: PropTypes.string,
-        rationale: PropTypes.node,
-        content: PropTypes.node,
-        correct: PropTypes.bool,
-        deselectEnabled: PropTypes.bool,
-        disabled: PropTypes.bool,
-        editMode: PropTypes.bool,
-        groupName: PropTypes.string,
-        isLastChoice: PropTypes.bool, // Needed for border styling
-        // This indicates the position of the choice relative to others
-        // (so that we can display a nice little (A), (B), etc. next to it)
-        // Also used to generate an id for each input.
-        pos: PropTypes.number,
-        reviewMode: PropTypes.bool,
-        showRationale: PropTypes.bool,
-        showCorrectness: PropTypes.bool,
-        type: PropTypes.string, // seems unused?
-
-        // Indicates whether the user has "crossed out" this choice, meaning
-        // that they don't think it's correct. This value does not affect
-        // scoring or other behavior; it's just a note for the user's
-        // reference.
-        crossedOut: PropTypes.bool,
-
-        // Indicates that the user has previously selected this answer. These
-        // answers may be rendered orange in review, rather than grey if
-        // incorrect.
-        previouslyAnswered: PropTypes.bool,
-
-        // A callback indicating that this choice has changed. Its argument is
-        // an object with two keys: `checked` and `crossedOut`. Each contains a
-        // boolean value specifying the new checked and crossed-out value of
-        // this choice.
-        onChange: PropTypes.func,
-
-        // Callbacks for when the user presses the up arrow or down arrow
-        // (respectively) to navigate between choices. When called, the parent
-        // finds the prev/next choice, focuses the input, and checks the radio
-        // button if appropriate. (The reason we have custom behavior is that
-        // we actually _don't_ want to auto-select crossed-out choices!)
-        goToPrevChoice: PropTypes.func.isRequired,
-        goToNextChoice: PropTypes.func.isRequired,
-    };
-
-    static defaultProps: $FlowFixMe = {
-        apiOptions: {
-            styling: {},
-        },
-        multipleSelect: false,
-        checked: false,
-        classSet: {},
-        correct: false,
-        disabled: false,
-        editMode: false,
-        onChange: function () {},
-        showRationale: false,
-        type: "radio", // seems unused?
-        pos: 0,
-        previouslyAnswered: false,
-    };
-
-    state: $FlowFixMe = {
-        isInputFocused: false,
-        isInputActive: false,
-    };
-
-    UNSAFE_componentWillUpdate(nextProps: $FlowFixMe) {
-        if (this.state.isInputFocused && nextProps.disabled) {
-            this.setState({
-                isInputFocused: false,
-            });
-        }
+function ChoiceIconWrapper(props: ChoiceIconWrapperProps) {
```

Comment:
This was just a mini component exclusively being used by Choice. It was small enough that I didn't see the point of giving it its own file.